src/rubocop/OffenseFormatter.js

```diff
@@ -0,0 +1,57 @@
+'use babel'
+
+import { satisfies } from 'semver'
+import getRuleDocumentation from './helpers/documentation-cache'
+
+import ErrorFormatter from '../ErrorFormatter'
+
+const SEVERITY_MAPPING = {
+  refactor: 'info',
+  convention: 'info',
+  warning: 'warning',
+  error: 'error',
+  fatal: 'error',
+}
+
+const HASCOPNAME_VERSION_RANGE = '>=0.52.0 <0.68.0'
+const RULE_MATCH_REGEX = /https:\/\/.*#(.*)/g
+
+export default class OffenseFormatter extends ErrorFormatter {
+  format(version, {
+    message: rawMessage, location, severity, cop_name: copName,
+  }, filePath) {
+    const hasCopName = satisfies(version, HASCOPNAME_VERSION_RANGE)
+    const [excerpt, url] = rawMessage.split(/ \((.*)\)/, 2)
```

Comment:
This seems _very_ fragile to me, what happens if there are `()` in the result message other than for the URL on the end?

Reply:
A Rubocop offense does not bring any property to get the ruby style guide URL, this is the only method we can use.

```
{
  "severity": "convention",
  "message": "Method has too many lines. [11/10] (https://rubystyle.guide#short-methods)",
  "cop_name": "Metrics/MethodLength",
  "corrected": false,
  "location": {
    "start_line": 75,
    "start_column": 3,
    "last_line": 87,
    "last_column": 5,
    "length": 380,
    "line": 75,
    "column": 3
  }
```